samples/full_example/full_example.js

```diff
@@ -1,63 +1,168 @@
-const getPaymentData = async() => {
-        
-    // Auth to get token
-    // ATTENTION: MAKE SURE NOT TO INCLUDE THIS AUTHENTICATION SNIPPET IN YOUR CLIENTSIDE APPLICATION
-    // THIS HAS BEEN DONE FOR DEMONSTRATION PURPOSES ONLY!!!!
-    const tokenRes = await fetch('https://<ENVIRONMENT_AUTH_URL>/auth/realms/toonie/protocol/openid-connect/token', {
+const baseUrl = "https://<ENVIRONMENT_API_URL>";
+const successUrl = "<SUCCESS_PAGE_URL>";
+const errorUrl = "<ERROR_PAGE_URL>";
+
+// Auth to get token
+// ATTENTION: MAKE SURE NOT TO INCLUDE THIS AUTHENTICATION SNIPPET IN YOUR CLIENTSIDE APPLICATION
+// THIS HAS BEEN DONE FOR DEMONSTRATION PURPOSES ONLY!!!!
+const getTokenData = async () => {
+    const tokenRes = await fetch(`${baseUrl}/auth/realms/toonie/protocol/openid-connect/token`, {
         method: 'POST',
         headers: {
             'Content-Type': 'application/x-www-form-urlencoded',
         },
         body: new URLSearchParams({
             "grant_type": "password",
-            "client_id": "pay-with-toonie",
-            "username": "customerusername",
-            "password": "customerpassword",
-        })
+            "client_id": "toonie-client",

```

Comment:
paywithtoonie-ext-client is the correct one iirc

Reply:
in demo we have `toonie-client`

Reply:
true, but not in prod that's what customers care about. Additionally, we need to standardise that in demo too as it is not correct that you are using the toonie app client for this.